test/k8sT/Services.go

```diff
@@ -1548,6 +1584,33 @@ var _ = Describe("K8sServicesTest", func() {
 						})
 					})
 
+					SkipContextIf(helpers.DoesNotExistNodeWithoutCilium, "Tests Maglev", func() {
+						var echoYAML string
+
+						BeforeAll(func() {
+							DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+								"global.tunnel":               "disabled",
+								"global.autoDirectNodeRoutes": "true",
+								"global.nodePort.algorithm":   "maglev",
+								"config.maglev.tableSize":     "251",
+							})
+
+							echoYAML = helpers.ManifestGet(kubectl.BasePath(), "echo-svc.yaml")
+							kubectl.ApplyDefault(echoYAML).ExpectSuccess("unable to apply %s", echoYAML)
+							err := kubectl.WaitforPods(helpers.DefaultNamespace, "-l name=echo", helpers.HelperTimeout)
+							Expect(err).Should(BeNil())
+						})
+
+						AfterAll(func() {
+							kubectl.Delete(echoYAML)
+						})
+
+						It("Tests", func() {
```

Comment:
```suggestion
						It("Tests LB consistency", func() {
```

Reply:
I've restructured the tests so that some of them could be run on 4.19 with `--node-port-algorithm=maglev`. Therefore, I've used different labels than your suggested one.